package.json

```diff
@@ -64,9 +64,10 @@
     "ava-init": "^0.1.0",
     "babel-core": "^5.8.23",
     "babel-plugin-espower": "^1.0.0",
+    "babel-runtime": "^5.8.29",
     "bluebird": "^3.0.0",
     "chalk": "^1.0.0",
-    "co": "^4.6.0",
+    "co": "git://github.com/floatdrop/co",
```

Comment:
```
"co": "floatdrop/co#343xsv324",
```

use the shorthand format and use a specific commit hash (first 6 chars of the hash is enough), otherwise the npm cache will become stale.